components/model/src/ome/model/internal/Permissions.java

```diff
@@ -205,10 +205,8 @@ public boolean isSet(Flag flag) {
 
     /**
      * returns the order of the bit representing the given {@link Flag}. This
-     * is dependent on the internal representation of {@link Permission} and
+     * is dependent on the internal representation of {@link Permissions} and
      * should only be used when necessary.
-     *
-     * @see ome.tools.hibernate.SecurityFilter
```

Comment:
http://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#see

```
@see "string"
```

would be ok.